kong/globalpatches.lua

```diff
@@ -232,6 +233,15 @@ return function(options)
         return decode_args_real(str, max_args or MAX_DECODE_ARGS, ...)
       end
       -- ]
+
+      -- READ REQUEST BODY [
+      _G.ngx.req.read_body = function()
+        -- for the same request, only one `read_body` call is needed
+        if not ngx.ctx.body_read then
```

Comment:
`body_read` is too common to conflict, could we give it a little long name?

Reply:
good point